packages/storage/src/indexed-db/index.ts

```diff
@@ -885,4 +885,24 @@ export class IndexedDb implements IndexedDbInterface {
       );
     });
   }
+
+  async getAssetTokenMetadata(
+    addressIndex: AddressIndex | undefined,
```

Comment:
If it's undefined, this function will always fail right? Is this intended?

Reply:
actually AddressIndex should never be undefined, since it will fail the transaction source [check](https://github.com/penumbra-zone/web/blob/main/packages/services/src/view-service/transaction-planner/index.ts#L70)?

Reply:
in that case, can we remove `undefined` from the type signature and use the non-null assertion operator on source?